### DISCUSSION TITLE
Duda sobre módulo "sumbit"

### DISCUSSION THREAD

**Discussion by iXrst** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#347] [Q&A]
Alguien sabe como hacer el módulo del submit que este sobrepuesto¿?
Yo estoy haciéndolo con "rows" y "containers", (para insertar imágenes y texto) pero no más no puedo sobreponerlo a la imagen.

**Comment by MesachVenegas** [comment]
puedes ajustar el nivel de los elemenots con z-index indicando cual es de mayor rango, 
aun estoy aprendiendo eso pero lo use para los rombos para que no se sobre ponga al navbar, solo le indico al contenedor del navbar un z-index: 3 y al elemto que no quiero que se superponga un z-index menor, en tu caso seria darle un valor mayor al modulo del submit y uno menor a la imagen
a se me olvidaba igualmente puedes colocarlo en la posicion si le estableces el position como absolute, y lo pocisionas usando top y left
